### PR TITLE
Fix to rollbar.js not loading in IE8

### PR DESCRIPTION
**Pull request by dlsteuer**

### CODE REVIEW

File: src/polyfill.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+function createPolyfills() {
+    // Object.create not supported in IE8 needs to be polyfilled
```
Comment (brianr) [inline src/polyfill.js]: Style: 2 spaces per level of indentation

**Comment by dlsteuer** [comment]
This will fix #205

**Comment by brianr** [comment]
As a possible alternative, is there a way we can change the code (i.e. what was added in 1.8) to not need to use Object.create?

**Comment by dlsteuer** [comment]
@brianr here is another way to do it, the browserstack tests are failing for IE8 still but unrelated to this, and this fix works in my testing in an IE8 browser.

File: src/xhr.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ function ConnectionError(message) {
   this.stack = (new Error()).stack;
 }
 
-ConnectionError.prototype = Object.create(Error.prototype);
+ConnectionError.prototype = new Error();
```
Comment (coryvirok) [inline src/xhr.js]: Looks like this thread explains some reasons why we shouldn't do this: http://stackoverflow.com/questions/783818/how-do-i-create-a-custom-error-in-javascript

**Comment by coryvirok** [comment]
If you put the logic for the retries into `XHR.post()` you can remove the direct enqueue logic.

**Pull request closed without merging**